javascript/javascript2/week1/main.js

```diff
@@ -0,0 +1,61 @@
+//Find the shortest word
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+function shortestWord(compare) {
+  return compare.reduce((a, b) => (a.length <= b.length ? a : b));
```

Comment:
wow, so clean ! Excellent use of .reduce 🏋️‍♂️ if I should comment on something than it would be the parameter naming, to make it even better one could avoid using verbs as variable names as it doesn't say much about what the argument is gonna be and they are much more apt to be great function names 🧚‍♀️

Reply:
Thank you, I work to become better at naming :)